CoreErrorReportView/src/au/gov/asd/tac/constellation/views/errorreport/ErrorReportDialog.java

```diff
@@ -0,0 +1,234 @@
+            default: errorIconColor = new Color(200, 120, 150);
+        }
+        final ImageView errorImage = new ImageView(UserInterfaceIconProvider.ERROR.buildImage(36, errorIconColor));
+        imageLabel.setGraphic(errorImage);
+        imageLabel.setPadding(new Insets(2 + (showOccs ? 4 : 0),0,0,0));
+        detailsBox = new VBox();
+        root.setTop(detailsBox);
+        
+        final BorderPane errorHeadingPane = new BorderPane();
+        final TextFlow errorHeadingText = new TextFlow();
+        final FlowPane headerSeverityPane = new FlowPane();
+        final Label severityDesc = new Label("Error Level: " + errorEntry.getErrorLevel().getName() + " ");
+        severityDesc.setPadding(new Insets(0,0,0,0));
+        final Label occurrenceDesc = new Label(" " + errorEntry.getOccurrences() + " Occurrences ");
+        headerSeverityPane.getChildren().add(severityDesc);
+        final Font outputFont = FontUtilities.getOutputFont();
+        final Text messageDesc = new Text(errorEntry.getHeading());
+        messageDesc.setStyle("-fx-font-family: " + outputFont.getFamily());
+        errorHeadingText.getChildren().add(messageDesc);
+        errorHeadingText.setPadding(new Insets(3,0,0,0));
+        
+        final BorderPane headingSection = new BorderPane();
+        
+        if (showOccs) {
+            headerSeverityPane.setPadding(new Insets(4,0,0,0));
+            occurrenceDesc.setTooltip(new Tooltip("Repeated Occurrences of this Exception"));
+            occurrenceDesc.setStyle("-fx-border-color:#808080; -fx-background-color: #e0e0e0");
+            occurrenceDesc.setTextAlignment(TextAlignment.CENTER);
+            occurrenceDesc.setPadding(new Insets(0));
+            final BorderPane occBox = new BorderPane();
+            occBox.setCenter(occurrenceDesc);
+            occBox.setStyle("-fx-background-color: #bcbcbc");
+            detailsBox.getChildren().add(occBox);
+        }
+        
+        headingSection.setCenter(headerSeverityPane);        
+        headingSection.setBottom(errorHeadingText);
+        errorHeadingPane.setLeft(imageLabel);
+        errorHeadingPane.setCenter(headingSection);
+
+        detailsBox.getChildren().add(errorHeadingPane);
+        errorMsgArea = new TextArea(errorEntry.getSummaryHeading() + "\n" + errorEntry.getErrorData());
+        errorMsgArea.setPrefRowCount(24);
+        errorMsgArea.setEditable(false);
+
+        summaryLabel.setText(" Click Show Details or see the messages.log file in your\n " + logFileLocation + " folder");
+        final ImageView blankImage = new ImageView(DefaultIconProvider.TRANSPARENT.buildImage(36, Color.RED));
+        summaryLabel.setGraphic(blankImage);
+        detailsBox.getChildren().add(summaryLabel);
+        final BorderPane buttonPane = new BorderPane();
+        buttonPane.setPadding(BUTTONPANE_PADDING);
+        root.setBottom(buttonPane);
+
+        showHideButton.setOnAction((ActionEvent event) -> toggleExceptionDisplay());
```

Comment:
either make lambda parameter final or remove explicit type (and brackets surrounding it)